src/main/java/uk/gov/justice/probation/courtcaseservice/application/WebClientConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.probation.courtcaseservice.application;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+@ConditionalOnProperty(name = "feature-flags.community-api-test-endpoint")
```

Comment:
@markberridge it occurred to me this was going to fall over as soon as we deployed to dev given we don't have valid credentials so wrapped this in a feature flag until such a time as we have them sorted in all envs. Also moved most of the config from application-local to application to prevent any missing properties causing it to fall over